kwc-challenge-card.html

```diff
@@ -9,26 +13,134 @@
1xculAZAYtwycGlE5UxsAiXWFy6URkFi3CJwWUCKuNgEa5ouExBZSQswhUOl0mojIVFuILhMg2V0bAIFx8uE1EZD4twFcZlKiorYBGu3LhMRmUNLMJVHpfpqKyCRbj+4bIBlXWwbCos39E7WCtbUFkJy1dcNqGyFpZvuGxDZTUsX3DZiMp6WK7jshWVE7BcxWUzKmdguYbLdlROwXIFlwuonINlOy5XUDkJy1ZcLqFyFpZtuFxD5TQsW3C5iMp5WKbjchWVF7BMxeUyKm9gmYbLdVRewTIFlw+ovIOlG5cvqLyEpQuXT6i8haUal2+ovIalCpePqLyHJRuXr6gIFlZA0r988RkVwfoflmhcvqMiWBmwROEiVP+KWlpaCiXJZLIsFotlldnPP0aBEeVe16pNsHIkGgZImHtcw5Q5H4KVJ90gUIK0dRkTweJMlwcMTxvOxznVjFasInEWgkOo8hePYHGsE7kAEarChSNYHLCyv4ogVMWLRrCK1yjdAkHhFY/HA9zlZ1OCFSB3gsVfLILFWavM7Y+2wuJFI1jFa5Tzf89IuOjwzkEnfxP6uiFc+WjFKlA3nlWJp024aOy+i2DRX+lIEUywcpQ1zCoU5h4piRrSKcHKCiIKkCj3GuJB2DAIVkYpRcAQ0YewdDV2RLD+L75IECL70mgj0qMJFv1jikiA8t3sPSyZq4vMvqVoENip17BUBK/iGQI9COvKW1gqA1f5LGEyInbkJSwdQet4ZkQbkW73DpbOgHU+O5KSEDd7BcuEYE0YQwgngW/xBpZJgZo0lsBiOG/wApaJQZo4Jk4zXM2ch2VygCaPjUtPgUZOw7IhOBvGGAaZs7BsCsymsfIicxKWjUHZOOZCyJyDZXNANo89G5lTsFwIxoU5IDJnYLkSCIbiwlycgOVCENlbie1zsh6W7QEUOgDbPDerYdlceN7XdlvnaC0sWwvOCyqznY1ztRKWjYUOA8pmXNbB8hFVCphNc7cKlk2FjbpC5bvflhpYA8uWgsoCZdu2aAUsQlWRq+k1MR6W6QVUsULZuC0aDYtQFWdrao2MhWVqwYpHrb6FibUyEpaJhVLPJdgTTauZcbBMK1CwePW2Nql2RsEyqTB6iYR/uik1NAaWKQUJH6k5d5pQSyNgmVAIc1iIGYnummqHpbsAYmI0sxedtdUKS+fEzaQgflS6aqwNlq4Ji4/O/B511FoLLB0TNT9+uSNUXXPlsFRPUG5cdvWusvZKYamcmF2RqxutqgyUwVI1IXUR2fskFVkogaViIvbGrGfksjORDkv2BPTE4sZTZWYjFZbMgbsRrf5ZyMpIGixZA9YfhXsjkJGVFFgyBupenGbNSHRmwmGJHqBZ5Xd7NCKzEwpL5MDcjtDc2YnKUBgsUQMyt+T+jExElkJgiRiIP7HZMdOomUaGFXUAdpTZz1FGyTYSrCgP9jMq+2YdNuPQsMI+0L7S0ojDZB0KVpgHUTx2VyBo5oFhBX2A3eWk0WdWIEj2gWAF6ZgiCV+Bffv2wYIFC2DdunUwZ86cdEfDhw+Hc+fOQaVKldKf7dixA6ZPnw5///6FRYsWwfHjx+HXr1/Qq1cvOHjwINSrV6/oQK5cuQKTJk2CeDwOJ06cSLffsGEDLF26FKpUqZL+bODAgXDq1Cn258OHD8OaNWvg/fv30KpVK9i1axd06dKF/Y4bFqEqmo+QBrNmzYLPnz/Ds2fPWNiZsHr37g3z5s2DcePGVXgWhrpnzx44f/481K1bF6ZMmcJAYPiFrqNHj8KWLVugXbt28PXr13Kwli9fDh8+fGBgsleuatWqQd++fRn0rl27sjYbN26Ep0+fQuXKlflgESohZrg6uXXrFnTr1g0GDRoEo0ePLgerQ4cOsHnzZhgyZEiFvnCFQpSIEa/nz58DtkekU6dOhdq1a8Pu3bvZ7yZPnsxWvUOHDsH9+/cZqvXr18O9e/fKwZo9ezbUqVMH1q5dW+F506ZNg5KSEjhw4ED6d82bN4cjR45Av379isMiVFwehDfKBSsWi0H37t3h8ePH8Pv3bxg1ahTbimrUqAH169dnqwduZ3iVlZVB9erVGZwmTZpA586d2Tb548cPBu3BgwcMW+pavXp1BVgTJkyAZDIJnz59gnfv3kGPHj1g06ZN0KJFCxgzZgz7L0JPXTjmsWPHMuAFt0JCJdwLd4e5YGFguB2OHz8eXr9+zULs06cPCxcR3b17F9q3b59+BsK5ePEi26rOnDkDixcvZiB37twJAwYMKDeWXLC2bt0KX758gblz57LVCbdhfMadO3dg6NChDDZiS2EeOXIkW60WLlyYHxah4jYgpWEuWNkPwoP2kiVL2LbXoEEDOHv2LEOEFx7mEdvDhw+hbdu27LOOHTuy8w+uYtlXLljZbXDVatSoEbx8+RLmz58PPXv2ZIhSVhArnv9mzpyZGxahkmIlUKfZsL5//w43b96E/v37p/s5duwYrFy5km2NeJDGt0M8P+H16NEjtm1+/PgRqlatCnv37k1vhXiwnzFjRtEV68aNG9C6dWu2zeKF2xtux2/evAF8G3316hU7p+GVSCQA31pPnjzJVtUKWyGhCpS/tMbZsPCNrVmzZrBq1SrAQzUGN2LECBg2bBj7bP/+/YBbF56z8MA9ceJEaNiwIfv8xYsXbHW5fv06/Pz5kyHEl4SWLVsWPGPhS0Ljxo1ZH3jhc/Gt7/Llywwz9nn69GkGGN8I8c0Qn4UvBuVgESppTrg6/vPnD9SsWZO1xbMQBoRbF64u27dvh2vXrrHvt548eQK1atViZy1EhVsebn3Lli1jb2X4PRZ+34SrFPaHqxy+YeK9eOEqd+HCBbh06RIMHjwYrl69Cvhs7ANXtzZt2rDtEs9xiAkh4VcXeJ7btm0bW7XwwhVzxYoV8PbtW+jUqRP7uuPbt2/szJWGhT+kDmFcVaBGVIE8FcAFCvGVJBKJspRCqhZVQEQFcLX7D/hqGjgMQrYYAAAAAElFTkSuQmCC"
```

Comment:
Do we want the default avatar to be part of this function, or could the default be a property on the element, that would allow us to potentially override this?

Reply:
Well, this is just a placeholder for if the component is used incorrectly